### PR TITLE
Changelog for TA P2

### PR DESCRIPTION
**Pull request by maririos**

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+- Sentiment label and sentiment scores are now part of `DocumentSentiment` and `SentenceSentiment`.
+- Renamed `TextSentiment` to `SentenceSentiment`.
+- Renamed `SentimentClass` to `SentimentLabel`.
+- Extandable ENUMs created for `Category`, and `SubCategory`.
+
+### New Features
+Credential class `TextAnalyticsApiKeyCredential` provides an `UpdateCredential()` method which allows you to update the API key for long-lived clients.
+
+### Fixes and improvements
+A new `HasError` property has added to `xxColletion` types to allor you to check if an operation on a particular document succeeded or failed.
```
Comment (kristapratico) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: some typos in this sentence

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+- `RecognizeEntitiesResult` property `NamedEntities` has been renamed to `Entities`.
+- `RecognizePiiEntitiesResult` now contains a list of `PiiEntity` instead of `NamedEntity`.
+- `RecognizePiiEntitiesResult` property `NamedEntities` has been renamed to `Entities`.
+- `RecognizeLinkedEntitiesResult` property `LinkedEntities` has been renamed to `Entities`.
+- `DetectLanguageResult` no longer has a property `DetectedLanguages`. Use `PrimaryLanguage` to access the detected language in text.
+- Created class `DocumentSentiment` that includes a list of `SentenceSentiment`.
+- Sentiment label and sentiment scores are now part of `DocumentSentiment` and `SentenceSentiment`.
+- Renamed `TextSentiment` to `SentenceSentiment`.
+- Renamed `SentimentClass` to `SentimentLabel`.
+- Extandable ENUMs created for `Category`, and `SubCategory`.
```
Comment (samvaity) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: Did this apply to all languages?

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md (hunk 1 above)
Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: 🤦‍♀

Comment (kristapratico) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: "xxCollection" too :)

Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: ohh that was intentional to say all types that end in Collection. Is there a better way to say it without enumerating all of them? Or should I just enumerate them lol

Comment (kristapratico) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: No, I'm fine with using "xx", just pointing out that your missing a "c" in collection

**Review by mssfang — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
 # Release History
 
+## 1.0.0-preview.2 (2020-02-11)
+### Breaking changes
+- The TextAnalyticsError model has been simplified to an object with properties `Code`, `Message`, and `Target`.
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: `TextAnalyticsError `

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+- Sentiment label and sentiment scores are now part of `DocumentSentiment` and `SentenceSentiment`.
+- Renamed `TextSentiment` to `SentenceSentiment`.
+- Renamed `SentimentClass` to `SentimentLabel`.
+- Extandable ENUMs created for `Category`, and `SubCategory`.
+
+### New Features
+Credential class `TextAnalyticsApiKeyCredential` provides an `UpdateCredential()` method which allows you to update the API key for long-lived clients.
+
+### Fixes and improvements
+A new `HasError` property has added to `xxColletion` types to allow you to check if an operation on a particular document succeeded or failed.
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: maybe use "`<document>Collection`" types make more sense?

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md (hunk 1 above)
Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: LOL!!!! 🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀🤦‍♀

**Pull request merged**